packages/grid/src/features/standard-table/components/StandardTableHeadItem.tsx

```diff
@@ -60,6 +61,7 @@ export const StandardTableHeadItem = React.memo(
       <th
         style={{
           background: background ?? "white",
+          borderTop: selectedBorder,
```

Comment:
Instead of styling the `th`, use the `selected` prop and pass it to the `TableHeadItem` which already has the styling.

You will probably also need to do something like this to allow the `TableHeadItem` to span the full height of the `td`:
```suggestion
          height: "100%",
```

Reply:
`TableHeadItem` doesn't fill the height of `th`. Adding `height: "100%"` should fix it.

![image](https://user-images.githubusercontent.com/6987939/214566524-65b923d7-f3a5-4199-8725-e7c3907afeb1.png)